src/utils/utils.cpp

```diff
@@ -327,6 +380,22 @@ std::string mp::utils::timestamp()
     return time.toString(Qt::ISODateWithMs).toStdString();
 }
 
+std::string mp::utils::match_line_for(std::istringstream& output, const std::string& matcher)
+{
+    output.seekg(0, output.beg);
```

Comment:
Feels weird to be rewinding the stream here… Should we make a copy of the stream instead? Or require it to be moved, maybe?

Reply:
I did this because of https://github.com/canonical/multipass/blob/c5c9f0c2df427023c788eabc16f2ebbe534b74c2/src/sshfs_mount/sshfs_mount.cpp#L61-L64

But yeah, after you mentioning it, I don't think `match_line_for()` should have a rewind.  I'll look into a better way to handle this.

Reply:
Fixed